cplusplus/UnconvertFromZDW.cpp

```diff
@@ -1878,12 +2047,51 @@ bool UnconvertFromZDWToMemory::OutputDescToFile(const string &outputDir)
 	char *sourceDir = NULL;
 	const char* outputBasename = NULL;
 	InitDirAndBasenameFromFileName(this->inFileName, sourceDir, outputBasename);
-	ERR_CODE eRet = this->outputDescToFile(this->columnNames, outputDir.c_str(), outputBasename, ".sql");
+	ERR_CODE eRet = this->outputDescToFile(this->columnNames, outputDir, outputBasename, ".sql");
 	if (sourceDir)
 		free(sourceDir);
 	return eRet == OK;
 }
 
+vector<std::pair<uint64_t,string> > UnconvertFromZDWToMemory::getFileLineage()
```

Comment:
nit: missing space after ','
also: this file already has "using std::pair", so perhaps remove the redundant "std::"?